services/src/main/resources/rule-es-mapping.json

```
@@ -0,0 +1,82 @@
+{
+  "properties": {
+    "id": {
+      "type": "long"
+    },
+    "title": {
+      "type": "text",
+      "index_options": "offsets",
+      "fields": {
+        "raw": {
+          "type": "keyword"
+        },
+        "whitespace": {
+          "type": "text",
+          "analyzer": "whitespace_lowercase_asciifolding"
+        }
+      }
+    },
+    "description": {
+      "type": "text",
+      "index_options": "offsets",
+      "fields": {
+        "raw": {
+          "type": "keyword"
+        },
+        "whitespace": {
+          "type": "text",
+          "analyzer": "whitespace_lowercase_asciifolding"
+        }
+      }
+    },
+    "audience": {
+      "type": "long"
+    },
+    "score": {
+      "type": "long"
+    },
+    "startDate": {
+      "type": "date",
+      "format": "epoch_millis"
+    },
+    "endDate": {
+      "type": "date",
+      "format": "epoch_millis"
+    },
+    "managers": {
+      "type": "long"
+    },
+    "domainId": {
+      "type": "long"
+    },
+    "area": {
+      "type": "text"
+    },
+    "event": {
+      "type": "text"
```

```suggestion
      "type": "keyword"
```
in case of constants and no full text search is needed, use `keyword` is more relevant